### PR TITLE
fix(@angular-devkit/core): fix typo for JobOutboundMessageKind.Pong

### PR DESCRIPTION
**Pull request by kyliau**
Marking this commit as `fix` instead of `refactor` as it's technically a
breaking change.

### CODE REVIEW

File: packages/angular_devkit/core/src/experimental/jobs/api.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ export enum JobOutboundMessageKind {
   OnReady = 'c',
   Start = 's',
   End = 'e',
-  Pong = 'p',
+  Ping = 'p',
```
Comment (alan-agius4) [inline packages/angular_devkit/core/src/experimental/jobs/api.ts:134]: 😅

**Review by alan-agius4 — APPROVED** [review]

File: packages/angular_devkit/core/src/experimental/jobs/api.ts (hunk 1 above)
Comment (clydin) [reply packages/angular_devkit/core/src/experimental/jobs/api.ts:134]: I’m not sure that’s a typo

Comment (alan-agius4) [reply packages/angular_devkit/core/src/experimental/jobs/api.ts:134]: I see, ping <-> pong

Comment (clydin) [reply packages/angular_devkit/core/src/experimental/jobs/api.ts:134]: For a network ping, ICMP uses echo request (type 8) and echo reply (type 0) for the messages.

**Comment by kyliau** [comment]
Closing this since "pong" here seems to imply an echo reply

**Pull request closed without merging**